pyopensn/src/aquad.cpp

```diff
@@ -0,0 +1,132 @@
+/*
+ * Created on Sun, February 23
+ *
+ * Copyright (c) 2025 The OpenSn Authors <https://open-sn.github.io/opensn/>
+ */
```

Comment:
These copyright notices should be using SPDX format, like [so](https://github.com/Open-Sn/opensn/blob/8dc54ff130546259027095aed84160b9e3aa03bc/framework/runtime.h#L1-L2)

Also, please, update `.licenserc.json` to check inside `pyopensn`